src/main/java/jenkins/plugins/slack/SlackNotifier.java

```diff
@@ -357,6 +357,7 @@ public boolean needsToRunAfterFinalized() {
     @Override
     public boolean perform(AbstractBuild<?, ?> build, Launcher launcher, BuildListener listener) throws InterruptedException, IOException {
         logger.info("Performing complete notifications");
+        listener.getLogger().println("[INFO] Slack notifications will be sent to the following channels: " + String.join(", ", this.getRoom()) + " on " + this.getTeamDomain());
```

Comment:
Much better, do you need the string.join? Isn’t it already a string?

Also teamDomain isn’t always set, you can use baseUrl instead.

Needs to either figure out which one is in use also checking which one is prioritised in the code or just only add the text if it’s set?

Reply:
tested this out locally and I get:
```
[INFO] Slack notifications will be sent to the following channels:  on 
```

![image](https://user-images.githubusercontent.com/21194782/50044443-53e4ed00-007b-11e9-8379-96ea85d6052e.png)

Reply:
The code isn't handling looking up from global config, see here:
https://github.com/jenkinsci/slack-plugin/blob/e3c060ac34b544f41271b2017f994cce77dc014b/src/main/java/jenkins/plugins/slack/SlackNotifier.java#L310